src/commands/e2e/ConfigTemplate.ts

```diff
@@ -24,6 +24,16 @@ export class ConfigTemplate {
                 browserName: '${browser}'
             }],`;
         }
+        let junitConfig = '';
+        if (jUnitReportPath) {
+            junitConfig = `, ['junit', {
+                outputDir: './${jUnitReportPath}',
```

Comment:
Instead of prefixing it with a local `./` here I would suggest we use `path.resolve(jUnitReportPath)` to ensure a fully qualified path (as such this will handle both relative and absolute paths). What happens if the directory doesn't exist yet? will it be created automatically?